### DISCUSSION TITLE
how to connect rdp via socks5(Username/Password Authentication) on MacOS.

### DISCUSSION THREAD

**Discussion by kwang87** [FreeRDP/FreeRDP#9739] [Q&A]
I try to command:

xfreerdp /u:user /p:'password' /title:test_rdp /cert-ignore /v:255.255.255.255 /proxy:socks5://proxyId:proxyPwd@proxyIP:proxyPort

its not working. no response.

how can i solve it?

**Comment by akallabeth** [comment]
need some more details to answer the question.
